Scoop/View/EditProfileViewController.swift

```diff
@@ -0,0 +1,256 @@
+        setupHeader()
+        setupProfileImageView()
+        setupUploadPhotoButton()
+        setupStackView()
+    }
+    
+    /// Temporary while save button is not implemented
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        
+        getUserPreferences()
+        updateUser()
+    }
+    
+    // MARK: - Initializers
+    
+    init(user: BaseUser, hometown: String) {
+        super.init(nibName: nil, bundle: nil)
+        self.user = user
+        if let profilePicUrl = user.profilePicUrl {
+            profileImageView.sd_setImage(with: URL(string: profilePicUrl))
+        } else {
+            profileImageView.image = UIImage(named: "emptyimage")
+        }
+        nameTextField.text = "\(user.firstName) \(user.lastName)"
+        pronounsTextField.text = user.pronouns
+        hometownTextField.text = hometown
+        classTextField.text = user.grade
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Setup View Functions
+    
+    private func setupHeader() {
+        let dottedLineMultiplier = 0.52
+        let solidLineVerticalInset = -12.75
+        let solidLineMultiplier = 0.32
+        let screenSize = UIScreen.main.bounds
+        let dottedline = UIImageView(image: UIImage(named: "dottedline"))
+        let solidline = UIView()
+        
+        dottedline.contentMode = .scaleAspectFit
+        view.addSubview(dottedline)
+        
+        dottedline.snp.makeConstraints { make in
+            make.height.equalTo(1)
+            make.width.equalTo(screenSize.width*dottedLineMultiplier)
+            make.trailing.equalToSuperview().inset(10)
+            make.top.equalTo(view.safeAreaLayoutGuide)
+        }
+        
+        solidline.backgroundColor = .black
+        view.addSubview(solidline)
+        
+        solidline.snp.makeConstraints { make in
+            make.height.equalTo(1)
+            make.width.equalTo(screenSize.width*solidLineMultiplier)
+            make.trailing.equalToSuperview()
+            make.top.equalTo(view.safeAreaLayoutGuide).inset(solidLineVerticalInset)
+        }
+        
+        navigationController?.navigationBar.prefersLargeTitles = false
+        navigationItem.title = "Edit Profile"
+        navigationController?.navigationBar.titleTextAttributes = [.font: UIFont(name: "Sen-Regular", size: 24)]
+    }
+    
+    private func setupProfileImageView() {
+        profileImageView.contentMode = .scaleAspectFill
+        profileImageView.clipsToBounds = true
+        profileImageView.layer.cornerRadius = 60
+        profileImageView.layer.borderColor = UIColor.scoopGreen.cgColor
+        profileImageView.layer.borderWidth = 3
+        view.addSubview(profileImageView)
+        
+        profileImageView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide).offset(35)
+            make.size.equalTo(120)
```

Comment:
Is it OK to make this a constant? Double check to see if it's weird w. smaller and bigger phones

Reply:
It looks like it works fine on other screen sizes